src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -41,6 +41,8 @@ import type {EventManager} from '../session/EventManager.js';
 import type {BrowsingContextStorage} from './BrowsingContextStorage.js';
 import type {CdpTarget} from './CdpTarget.js';
 
+import ResultOwnership = Script.ResultOwnership;
```

Comment:
This looks wrong as it uses import rather then `const` - not sure how that works.
But I am also not sure the benefit of having a shorter name in a single file gives us.

Reply:
It was IDE's auto import. Inlined.